terraform/environments/nomis/ec2-jumpserver.tf

```diff
@@ -13,13 +13,13 @@ data "github_team" "this" {
   slug = "studio-webops"
 }
 
-data "aws_subnet" "private_az_a" {
+data "aws_subnet" "this" {
```

Comment:
We'd want to look up 2 subnets to make the ASG cross-az. So I think this needs to be changed back and another data lookup for aws_subnet in`{local.region}b`

Reply:
Roger that